backend/setupStaticContent.js

```diff
@@ -7,9 +7,10 @@ module.exports = () => {
   router.use(express.static(path.join(process.cwd(), '/build')))
 
   const assetPaths = [
```

Comment:
Updated paths for the new location of govuk-frontend asssets (within a /dist sub-folder)